### ISSUE TITLE
On User Create

### ISSUE THREAD

**Issue by kylepolich** [data-skeptic/fast-spa#145]
- [ ] Create in AppWrite
- [ ] Copy `sys/msgs/welcome.md` to `user/{username}/apps/forager/welcome.md`
- [ ] Add a link to the user's Homepage #139 for a Forager App which has a root of `user/{username}/apps/forager/`
- [ ] One-time script to migrate DCS users to AppWrite

**Comment by katiarabelo** [comment]
➤ Cory Armstrong commented:

I've extended my original estimate on this ticket, because it is technically two tickets in one.

The first few steps of this involve fine-tuning the implementation of AppWrite in general. The original setup was done quickly to have a working model, and see that the tool works for what we need it for, however with the way that it was setup, the steps in this ticket are not possible without various "hacky" solutions.

Another ticket should be created that allows me to finalize the login/registration setup of this with AppWrite so that this can be possible.

**Comment by katiarabelo** [comment]
➤ Cory Armstrong commented:

Waiting on #188 ⁃ Finalize the login/registration setup ( https://app.asana.com/0/1120876312410440/1158608334497966/f )
